src/bowling.test.js

```diff
@@ -0,0 +1,69 @@
+import Bowling from './bowling.js';
+
+let bowling;
```

Comment:
The jest tests are so much clearer than Jasmine! Is that why you chose to change? It looks so much more like RSpec to me.

Reply:
Yes, it looks more like RSpec!